packages/artillery/lib/cmds/run-fargate.js

```diff
@@ -99,13 +99,11 @@ RunCommand.flags = {
   }),
   cpu: Flags.string({
     description:
-      'Set task vCPU on Fargate. Value may be set as a number of vCPUs between 1-16 (e.g. 4), or as number of vCPU units (e.g. 4096)',
-    default: '4'
+      'Set task vCPU on Fargate (default 4vCPU). Value may be set as a number of vCPUs between 1-16 (e.g. 4), or as number of vCPU units (e.g. 4096).'
```

Comment:
```suggestion
      'Set task vCPU on Fargate (defaults to 4 vCPU). Value may be set as a number of vCPUs between 1-16 (e.g. 4), or as number of vCPU units (e.g. 4096).'
```

Reply:
Space between value & unit.

"(default 4 vCPU)" looks odd, went for "(defaults to 4 vCPU)" vs "(default: 4 vCPU)" here